Apps/Database/src/Delegates/DbResourceDelegateDelegate.cs

```diff
@@ -208,7 +208,7 @@ public async Task<ResourceDelegateQueryResult> SearchAsync(ResourceDelegateQuery
                 dbQuery = dbQuery.Take(query.TakeAmount.Value);
             }
 
-            IEnumerable<ResourceDelegate> items = await dbQuery.ToArrayAsync().ConfigureAwait(true);
+            IList<ResourceDelegate> items = await dbQuery.ToArrayAsync().ConfigureAwait(true);
```

Comment:
```suggestion
            IList<ResourceDelegate> items = await dbQuery.ToArrayAsync();
```

ConfigureAwait is no longer required.

https://devblogs.microsoft.com/dotnet/configureawait-faq/ has a great break down of the reason we would want to use the ConfigureAwait(false), however he (the author) also states: 
> Why would I want to use ConfigureAwait(true)?
You wouldn’t, unless you were using it purely as an indication that you were purposefully not using ConfigureAwait(false) (e.g. to silence static analysis warnings or the like). ConfigureAwait(true) does nothing meaningful. When comparing await task with await task.ConfigureAwait(true), they’re functionally identical. If you see ConfigureAwait(true) in production code, you can delete it without ill effect.
> 
> The ConfigureAwait method accepts a Boolean because there are some niche situations in which you want to pass in a variable to control the configuration. But the 99% use case is with a hardcoded false argument value, ConfigureAwait(false).

Reply:
I didn't introduce this one but it can definitely be removed.

Reply:
Removed.